### PR TITLE
Add disaster key encryption and fix cluster deployment for dc6-proto1

### PR DESCRIPTION
**Pull request by celskeggs**
There are lots of changes here.

### CODE REVIEW

**Comment by celskeggs** [comment]
@ichung: did this get reviewed at the hackathon? do you expect this to be reviewed by the end of today, or should I just merge it?

**Comment by ichung** [comment]
@celskeggs I didn't finish reviewing it, but I'll have done so by end of day

File: building/build-debs/homeworld-keysystem/src/keygen/generate.go [hunk 1]
```diff
@@ -24,7 +24,7 @@ func GenerateTLSSelfSignedCert(key *rsa.PrivateKey, name string, present_as []st
 
 	certTemplate := &x509.Certificate{
 		KeyUsage:    x509.KeyUsageDigitalSignature | x509.KeyUsageCertSign,
-		ExtKeyUsage: []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth},
+		ExtKeyUsage: []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth, x509.ExtKeyUsageClientAuth},
```
Comment (ichung) [inline building/build-debs/homeworld-keysystem/src/keygen/generate.go]: To what is this certificate used to authenticate as a client?

Comment (celskeggs) [reply building/build-debs/homeworld-keysystem/src/keygen/generate.go]: AHAHAHAHAHAHA

it's not. but things that it *issues* need to authenticate as clients, which apparently means this needs to be here.

File: building/build-debs/homeworld-admin-tools/src/authority.py [hunk 2]
```diff
@@ -28,18 +28,31 @@ def generate() -> None:
...
             else:
                 raise e
-        print("packing authorities...")
-        subprocess.check_call(["tar", "-C", certdir, "-czf", authorities, "."])
+        print("encrypting authorities...")
+        cryptdir = os.path.join(d, "cryptdir")
+        os.mkdir(cryptdir)
+        for filename in os.listdir(certdir):
+            if filename.endswith(".pub") or filename.endswith(".pem"):
+                # public keys; copy over without encryption
```
[the remark this answers is not in the document]
Comment (celskeggs) [reply building/build-debs/homeworld-admin-tools/src/authority.py]: Split into #144.

**Pull request merged**